public/components/StatusPanel.vue

```diff
@@ -272,10 +292,12 @@ export default Vue.extend({
         });
 
       // serialize the modes map into a string and add to the route
+      // and update to know that the clustering has been applied.
       const varModesStr = varModesToString(varModesMap);
       const entry = overlayRouteEntry(this.$route, {
         varModes: varModesStr,
         dataMode: DataMode.Cluster,
+        clustering: "1",
```

Comment:
Should this be defined as an enum?

Reply:
It is possible, maybe we can refactor that when we redo the route in our blue sky project.

The current choice was made quickly to avoid refresh of the _Clustering_ in a [previous issue.](https://github.com/uncharted-distil/distil/pull/2223/files?file-filters%5B%5D=.go&file-filters%5B%5D=.ts#diff-289d9418bc0c4feaf33efa893939615725e09ae0fc202000f318f43eb8ffcc0eR197)